core/trino-main/src/main/java/io/trino/sql/planner/Symbol.java

```diff
@@ -50,6 +56,11 @@ public SymbolReference toSymbolReference()
         return new SymbolReference(name);
     }
 
+    public io.trino.sql.ir.SymbolReference toIrSymbolReference()
```

Comment:
Remove package name. This class should not have any dependencies on the AST, so there shouldn't be any conflicts.